src/lib/populatePresetParams.ts

```diff
@@ -0,0 +1,18 @@
+import { Params } from '../config/params';
+import { Step } from '../types/Step';
+import { StepConfig } from '../types/StepConfig';
+
+// Populate steps with params from request.
+export default (
+  steps: Step[],
+  params: Params,
+): Step[] => {
+  return steps.map((step: Step) => {
+    const name = Object.keys(step)[0];
+    const config: StepConfig = Object.keys(step[name]).reduce((acc, key) => ({
+      ...acc,
+      [key]: params[key] || step[name][key],
```

Comment:
Does the path parameter has to have the same name as the property its replacing? I don't see any reference to the value stated in `from_param`.

Reply:
I'm looking in to it now.

Reply:
Propsed changes can be seen here #42